client-templates/gitlab/.env.sample

```diff
@@ -23,3 +23,13 @@ ACCEPT=accept.json
 
 # The path for the broker's internal healthcheck URL. Must start with a '/'.
 BROKER_HEALTHCHECK_PATH=/healthcheck
+
+# the host where the git server resides
+GIT_URL=$GITLAB
+
+# git credentials for cloning repos
+GIT_USERNAME=oauth2
```

Comment:
Where did you get those usernames?

Reply:
I think those usernames are standard ones in cases when there is only a user token (or equivalent) used for a specific SCM.

@shinin-light is that right?

Reply:
Sadly there is no standard across the SCMs for the git protocol. This specific username comes from the gitlab specifications, but it should be tested with a real gitlab environment (which I didn't). I didn't find any reasonably quick way of testing the broker with every single SCM integration we support at the moment, but if there is one we should try it out.

Reply:
Okay I understand.
Isn't just following how to locally set up a brokered integration, while using these vars will be enough for that purpose?

Reply:
@ohad2712 You mean testing different SCMs?

Reply:
Yes.

Reply:
Makes sense. We are working on testing, will get back to you once we gone through all of the SCMs.